test/src/main/java/mil/nga/giat/geowave/test/mapreduce/MapReduceTestEnvironment.java

```diff
@@ -54,20 +57,27 @@ protected void testMapReduceIngest(
 		String[] args = null;
 
 		// Indexes
-		IndexPluginOptions indexOption = new IndexPluginOptions();
-		indexOption.selectPlugin(dimensionalityType.getDimensionalityArg());
-
+		String[] indexTypes = dimensionalityType.getDimensionalityArg().split(
```

Comment:
It would be nice to make this a static method in IndexPluginOptions because it will probably be used elsewhere ...